src/lib/client/AuthorityClient.ts

```diff
@@ -0,0 +1,82 @@
+import type { Command } from '../commands/Command.js';
+import type { Request, RequestBody } from '../utils/requests.js';
+import { HTTP_STATUS_CODES } from '../utils/http.js';
+
+import { ServerError } from './ServerError.js';
+import type { AuthorizationHeader } from './AuthorizationHeader.js';
+import { ClientError } from './ClientError.js';
+
+const USER_AGENT = 'VeraId Authority Client (https://github.com/relaycorp/veraid-authority-js)';
+const JSON_CONTENT_TYPE = 'application/json';
+
+interface BadRequestResponseBody {
+  type?: string;
+  message?: string;
+}
+
+export class AuthorityClient {
+  protected readonly baseUrl: string;
+
+  protected readonly authHeader: string;
+
+  public constructor(baseUrl: string, authHeader: AuthorizationHeader) {
+    // eslint-disable-next-line regexp/no-super-linear-move
+    this.baseUrl = baseUrl.replace(/\/+$/u, '');
```

Comment:
I was not able to find a workaround to this. So I guess disabling the rule will do